ticf-cri-stub/src/test/java/uk/gov/di/authentication/ticf/cri/stub/lambda/TICFCRIStubHandlerTest.java

```diff
@@ -2,20 +2,55 @@
 
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.google.gson.Gson;
 import org.junit.jupiter.api.Test;
+import uk.gov.di.authentication.ticf.cri.stub.lambda.entity.TICFCRIRequest;
+import uk.gov.di.authentication.ticf.cri.stub.lambda.entity.TICFCRIStubResponse;
+
+import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.mock;
 
 class TICFCRIStubHandlerTest {
+    private static final Context context = mock(Context.class);
+
     @Test
-    void shouldReturn200ForSuccessfulRequest() {
-        var handler = new TICFCRIStubHandler();
-        var context = mock(Context.class);
+    void shouldReturn200ForSuccessfulValidRequest() {
+        TICFCRIStubHandler handler = new TICFCRIStubHandler();
         var event = new APIGatewayProxyRequestEvent();
 
+        String testInternalPairwiseId = "urn:fdc:gov.uk:2022:test";
+        List<String> testVtr = List.of("Cl.Cm");
+        String testJourneyId = "44444444-4444-4444-4444-444444444444";
+        Boolean testAuthenticated = true;
+        List<String> testCi = List.of("D03", "F01");
+        String testInterventionCode = "01";
+        String testInterventionReason = "01";
+
+        TICFCRIRequest ticfcriRequest =
+                new TICFCRIRequest(
+                        testInternalPairwiseId,
+                        testVtr,
+                        testJourneyId,
+                        testAuthenticated,
+                        false,
+                        false);
+        Gson gson = new Gson();
+        String requestJson = gson.toJson(ticfcriRequest);
+        event.setBody(requestJson);
         var result = handler.handleRequest(event, context);
+
+        TICFCRIStubResponse ticfcriStubResponse =
+                new TICFCRIStubResponse(
+                        testInternalPairwiseId,
+                        testJourneyId,
+                        testCi,
+                        testInterventionCode,
+                        testInterventionReason);
+        String responseJson = gson.toJson(ticfcriStubResponse);
```

Comment:
We have a shared test helper `hasJsonBody` that might be helpful here? Or sometimes I like being able to read the json as a string, so I wouldn't mind also switching this to be a json string. But might look ugly, I'll let you decide.